lua/autorun/client/cl_chip_lister.lua

```diff
@@ -257,6 +257,8 @@ local function updateListDraw( plyCount, globalUsage, perPlyData )
     for i = 1, plyCount do -- Draw the info of each owner and their chips
         local data = rawget( perPlyData, i )
 
+        if not data then break end -- Sometimes some addon manages to bug out hard enough to empty out our table. Needs to be looked into, but this will avoid an error for the time being.
+
         elemCount, x, xEnd, y = drawPlayersChipData( data, elemCount, x, xEnd, y )
     end
```

Comment:
why not just loop over perPlyData

Reply:
instead of keeping track of plyCount with a net message you could just get the length of the table when its received.

Reply:
I can switch it to that, thoughnow we have to find out what is breaking the table to begin with. Maybe something is breaking JSON formatting for the net message?